lib/promote_release.js

```diff
@@ -321,7 +321,7 @@ class ReleasePromotion {
     // Since we've committed the Working On commit, the release
     // commit will be 1 removed from tip-of-tree (e.g HEAD~1).
     const releaseCommitSha = this.getCommitSha(1);
-    runSync('git', ['checkout', 'master']);
+    runAsync('git', ['checkout', 'master']);
```

Comment:
```suggestion
    runSync('git', ['checkout', 'master']);
```

Or convert the method to be async